exporter.go

```diff
@@ -25,7 +25,7 @@ type WmiCollector struct {
 }
 
 const (
-	defaultCollectors = "cpu,cs,logical_disk,net,os,system"
+	defaultCollectors = "cpu,cs,logical_disk,net,os,service,system"
```

Comment:
I'd say so, yes, but I'm open to changing it. It adds ~400 metrics on my systems, give or take, which doesn't feel that heavy. The IIS collector creates around ~500 on our web servers. On the other hand, it would still be an increase of the total with about 40%. 

I know I'll enable it everywhere, but that doesn't mean everyone else wants it :) I'm fine either way.